README.md

```diff
@@ -61,6 +61,7 @@ Backend servers:
   - `dynamic`: If `True` lookup IP on every request, default `False` (only lookup at startup).
   - `cache_validity`: The time that an object should be cached for, if omitted caching is disabled for this backend
   - `websockets`: If `True` enable proxying of websockets, default `False`
+  - `websocketsonly`: If `True` only allow websocket requests, otherwise return HTTP status 400, default `False`
```

Comment:
In the current implementation, it might be worth stating that this parameter only applies if `websockets` is set to `True`